src/ray/gcs/gcs_client/accessor.cc

```diff
@@ -1294,5 +1294,20 @@ Status InternalKVAccessor::Exists(const std::string &ns,
   return ret_promise.get_future().get();
 }
 
+RuntimeEnvAccessor::RuntimeEnvAccessor(GcsClient *client_impl)
+    : client_impl_(client_impl) {}
+
+Status RuntimeEnvAccessor::PinRuntimeEnvUri(const std::string &uri,
+                                            int expiration_s,
+                                            int64_t timeout_ms) {
+  rpc::PinRuntimeEnvURIRequest request;
+  request.set_uri(uri);
+  request.set_expiration_s(expiration_s);
+  rpc::PinRuntimeEnvURIReply reply;
+  auto status =
+      client_impl_->GetGcsRpcClient().SyncPinRuntimeEnvURI(request, &reply, timeout_ms);
+  return status;
```

Comment:
Any reason we skipped checking and updating `status` as in `PythonGcsClient::PinRuntimeEnvUri()`?

Reply:
GcsRpcClient method already checks the `status` within the proto reply: https://github.com/ray-project/ray/blob/master/src/ray/rpc/gcs_server/gcs_rpc_client.h#L211